src/autolabel/data_loaders/validation.py

```diff
@@ -237,9 +241,11 @@ def validate(self, data: List[dict]) -> List[Dict]:
     def validate_dataset_columns(self, dataset_columns: List):
         """Validate columns
 
-        Valiate if the columns mentioned in example_template dataset are correct
-        and are contined within the columns of the dataset(seed.csv)
+        Validate if the columns mentioned in example_template dataset are correct
+        and are contained within the columns of the dataset(seed.csv)
         """
+        if self.explanation_column in dataset_columns:
+            dataset_columns.remove(self.explanation_column)
```

Comment:
this will modify the list being passed into the function - might be difficult for the caller to understand

Is it possible to remove this code from validate_dataset_columns and let expected_columns handle the removal of explanation column?